### DISCUSSION TITLE
Error opening URL at Windows

### DISCUSSION THREAD

**Discussion by nji9nji9** [audacious-media-player/audacious#1407] [Q&A] [answered]
When I try to open an URL I always get the message...
The text is in German (I haven't found a way to change the language in the settings).
Translation:
"Error playing of https://st01.sslstream.dlf.de/dlf/01/128/mp3/stream.mp3?aggregator=web
Could not connect to server: the access to a socket was not allowed
due to access rights of the socket."

Activating "SOCKS proxy" doesn't help.

Audacious v. 4.3.1 @ Win7-64 SP1

TIA

**Comment by radioactiveman** [comment]
The error could be related to the HTTPS certificate (see https://github.com/audacious-media-player/audacious/issues/1248). Audacious 4.4  trusts all certificates which Windows itself trusts. So please try the new version, it should be available soon. Until then you may also use the [beta release](https://audacious-media-player.org/download).

Edit: Ah, you use Windows 7. Then 4.4 will probably not work anymore. :(

**Reply by nji9nji9** [reply]
Thankyou for your reply.
I assumed too about HtTPS certificate.
But I get the same message with
http://snd.radio-z.net:8000/Radio-Z

BTW:
Can you tell me how to change language (see first post) too, please?

**Reply by radioactiveman** [reply, marked as answer]
Audacious uses the same UI language as Windows. You could change the language of Windows, but this won't help us here.
Have you tried the 4.4 version? Maybe it still works on Windows 7, even though it is unsupported.

@jlindgren90: Is the Windows 10 requirement caused by Qt 6?

**Reply by jlindgren90** [reply]
Yes, see https://doc.qt.io/qt-6/windows.html#supported-configurations

**Reply by nji9nji9** [reply]
Thank you for the help.
Actually I'm about to switch to Linux, and do a check which apps to use there.
For foobar2000 it would be Audacious for sure.
I thought that maybe it could replace streamWriter for listening and capturing.
Listening with it would be beneficial due to crossfeed (for my headphones).
But obviously Audacious can't _capture_ a stream (only copy and encode),
and moreover only the stream playing. So I'll have to find an app for
audiostream capturing (which isn't blown up :) ).
Concerning UI language:
I definitely would suggest to make it changeable by UI. Everything is at hand.

Thankyou for my new player at Linux! :)
